### DISCUSSION TITLE
Team Division Poll for Milestone 1

### DISCUSSION THREAD

**Discussion by rukitoyordan** [uprm-inso4101-2023-2024-s2/semester-project-rumaradoptionapp#2] [Polls]
Good afternoon everyone! We require active participation from every member in order to ensure our project runs smoothly into its first stages of development. There will be **6 teams consisting of 4 people each** divided into the Main Features that were discussed in class last Friday (2 February 2024). Below is a bit of information on what we expect from each team and how they synergize. Some teams will need to communicate constantly and work sometimes as an 8 person team if need be.

### Teams to Choose From (only 4 per Team) 🚻

1. **Website Creation**: We have decided on making a website rather than an app. Members should research on HTML, CSS, Javascript and try to come up with ideas on website creation. This team will work the most at the beginning and try to find ways to connect the rest of the features. (Website should try having a test realm and an official realm)

2. **Sign-Up/Login Feature**: This team will work on creating a sign-up and login feature for creation of new profiles and access. Members must research on tasks regarding this and how to store as well as access profile info like usernames/passwords. This team will be partnered alongside the Website Creation team.

3. **Human Profiles**: This team will work on how to display, connect, and modify profiles after sign-up. It is important to know how to store this information as well as display it on the website. This team should work with Sign-Up/Login when the time comes to privately store data.

4. **Pet Registration**: Similar to Sign-Up/Login but only for pets. Pets should only be able to be added once you have a human profile made. Pets should be able to be added with details, pictures, etc.

5. **Pet Listings**: Area in the website where pets will be shown and displayed BRIEFLY. You have creative freedom on this, but we thought on how maybe social media scrolling with sections for example like Fosters or Adoptable Pets. Website Creation Team should know about your ideas to make sure the website looks seamless. (Main focus will be feature first, then decoration)

6. **Pet Profiles**: Once a pet is registered (Pet Registration), they should have their own profile similar to Human Profile. These can be edited for now by anyone but should later on be modifiable by people fostering them or directive members of RUMAR. The specifics of the pet should be displayable in detail, unlike Pet Listings which would show brief info.

### Important Notes 📝
The purpose of this poll essentially is to have everyone divide and pick **ONE** team to be a part of. Additionally, once the teams are chosen **ONE** member of each team will become a Team Leader.

The job of the Team Leader will be to document everything that your team does, as well as have their own jobs alongside developers to create tasks and make sure they are done in a timely manner. More information will be announced soon on possible weekly Team Leader Meetings (Managers and Team Leaders only) once we get this all started.

Developers start discussing your ideas for tasks in a separate section I will make. If you don't choose a team by the deadline, you will be placed in a random team. Do not hesitate in contacting me or @evnd04 if you have any questions or doubts!

**_The deadline to answer this poll will be Monday February 5th, 2024 @ 11:59 PM_** ❗️

**Comment by frances-sepulveda** [comment]
Frances Sepulveda-Sign up/log in Feature :)

**Comment by rukitoyordan** [comment]
Please make sure that after you pick to comment below what team you chose. Since we are unable to see who exactly picked each vote. Thanks @frances-sepulveda for your input on this!

Example:

Fabián A. Pérez Muñoz - Sign-Up/Login

**Comment by JohnSantiagoRuiz** [comment]
John P. Santiago Ruiz - Pet Registration

**Comment by diegorobles2** [comment]
Diego O. Robles Amengual - Pet Listings

**Comment by smbv** [comment]
Sofia M. Barrera Vega - Website Creation

**Comment by Adriana-Isabel** [comment]
Adriana I. Carrasquillo Cruz - Website Creation

**Comment by GiovanniMaysonet** [comment]
Giovanni Maysonet Rodriguez
Web Creation

**Comment by christianperez42** [comment]
Christian J Perez- Website creation

**Comment by rukitoyordan** [comment]
**NOTICE:** Website Creation Team is Full! 🚨
Final Members are posted in the Website Creation Team category.

**Comment by yamayrie** [comment]
Yamayrie Lebron – Sign-up/login feature

**Comment by Saul-Caban-UPRM** [comment]
Saúl E. Cabán Torres-sign-up/login feature

**Comment by Javi510348** [comment]
Javier Torres Rivera - Sign-Up/login feature

**Comment by frances-sepulveda** [comment]
Hello Everyone! For the Sign-up/Log in Feature I would like being the team lead! Please let me know if anyone else in the team would want the position as well so we can talk a bit more about it :)

**Reply by Javi510348** [reply]
I haven't been confirmed as part of the sign-up/login team, but I have no issue with you being the team lead.

**Reply by frances-sepulveda** [reply]
> I haven't been confirmed as part of the sign-up/login team, but I have no issue with you being the team lead.

Thank you :)

**Comment by Angel-Rosado** [comment]
Angel Rosado - Human Profiles

**Comment by Alejandro-DeJesus** [comment]
Alejandro J. De Jesus - Pet Profiles

**Comment by rukitoyordan** [comment]
NOTICE: Sign-Up/Login Team is Full! 🚨
Final Members are posted in the Sign-Up/Login Team category.

**Comment by RaquelQuinones** [comment]
Raquel Quinones - Pet Profiles

**Comment by rafa-quiles** [comment]
Rafael E. Quiles - Human Profiles

**Comment by cxprss** [comment]
Hector G. Ramos Cortes - Pet Listings

**Comment by Christian2101** [comment]
Christian N. Rodriguez Figueroa - Pet Listings

**Comment by rukitoyordan** [comment]
Friendly reminder for those who haven't joined a team yet. Deadline is tonight **Tuesday February 5th, 2024 @ 11:59PM**. If you are unsure of which team to join discuss it with us so you aren't placed in a random team.

Have a lovely day!💕

**Comment by Alej24** [comment]
Alejandro J. Rivera Batista - Pet Listings

**Comment by rayanemili** [comment]
Rayane Mili - Pet Profiles

**Comment by ErickCruz4** [comment]
Erick Y Cruz Schaeffer - Pet Registration

**Comment by HectorSepul** [comment]
Hector R. Sepulveda - Pet Registration

**Comment by AleDiazz** [comment]
Alejandro Diaz - Pet Profiles

**Comment by frances-sepulveda** [comment]
Hello everyone! 

This is the Sign-up/login team group chat, 
Please join when you can to set up meetings, etc! Thank you :)

https://chat.whatsapp.com/B5reNueAFwM7opPXdIlAyw

**Reply by RoBdIeL** [reply]
Algun chat para el Pet Registration??
